spec/controllers/api/irs_attempts_api_controller_spec.rb

```diff
@@ -107,6 +79,72 @@
         expect(Base64.strict_decode64(response.headers['X-Payload-Key'])).to be_present
         expect(response.body).to eq(test_object)
       end
+
+      context 'with aws_s3_stream enabled' do
+        let(:test_object) { '{test: "1234567890 12345"}' }
+        before do
+          allow(IdentityConfig.store).to receive(:irs_attempt_api_aws_s3_stream_enabled).
+            and_return(true)
+          allow(IdentityConfig.store).to receive(:irs_attempt_api_aws_s3_stream_buffer_size).
+            and_return(10)
+
+          Aws.config[:s3] = {
+            stub_responses: {
+              head_object: { content_length: test_object.bytesize },
+              get_object: proc do |context|
+                range_string = context.params[:range]
+                byte_string = range_string.split('=')[1]
+                byte_range_array = byte_string.split('-')
+                { body: test_object.byteslice(
+                  byte_range_array[0].to_i,
```

Comment:
I think I understand what this is doing, but I didn't know you could do `_` in place of a variable name? Is that common syntax? I am assuming it is basically saying "We don't care about this part". 

Or is it literally assigning that part to a variable named `_`?

Reply:
in the REPL, `_` is like the last assigned value. In a script, `_` is a throwaway name. Any name starting with an underscore is a convention for "I am assigning this but I won't use it" because here it's basically a placeholder

Reply:
Thanks! I will make this change